### Crpyto Mining Electricity Consumpation (ETH) VS WorldWide Electricity Consumption

So I keep hearing about Crypto Mining Electricity Consumption in the news, so I thought it would be fun to actually find out what is the reality of it.

All the data used here is publicly available.


ETH Hashrate : 860.73 TH/s at 07-01-2022 11:23 AM (Source : https://2miners.com/eth-network-hashrate)

RTX 3060 will be used for calculating energy consumption.
RTX 3060 ETH mining stats: 50 MH/s at 120 W (Source : https://miningchamber.com/gpu-mining/rtx-3060-mining-settings/)

Total Electricity Consumption Worldwide: 23,398,000 GW.h/yr (Source: https://en.wikipedia.org/wiki/List_of_countries_by_electricity_consumption)


So, we have to calculate the total power consumed by mining worldwide (ETH).


In [22]:
hashrate_3060 = 50
power_3060 = 120
eth_hashrate = 860.73 * 1000 * 1000 #converting TH/s to MH/s
eth_power = (power_3060/hashrate_3060) * eth_hashrate #calculating total power by taking RTX 3060 as reference
print("Total power consumed by ETH mining worldwide(in W): " + str(eth_power))

Total power consumed by ETH mining worldwide(in W): 2065752000.0


Now, converting Total Electricity Consumption to W

In [21]:
total_elc = 23398000 * 10**9 #converting GW to W
total_pow = total_elc/(24*365) #dividing energy by time to get power
print('Total power consumed worldwide(in W): '+ str(total_pow))

Total power consumed worldwide(in W): 2671004566210.046


Percentage of power consumed by ETH mining compared to Total power consumption:

In [16]:
per = eth_power/total_pow * 100
print(per)

0.07733989024702966


So, we conclude that crypto mining for Ethash(ETH) consumes only about 0.077% electricity consumed worldwide. If we take into account other mining algorithms and add them, it will still be less than 5x what Ethash consumes. 

In [19]:
per_total = per * 5

In [20]:
print(per_total)

0.38669945123514826


So, we observe that total electricity consumption for all mining algorithms(approximately) is only 0.39% of the total consumption worldwide.

This was made by

Akshit Singh

KJ Somaiya College Of Science And Commerce, Mumbai